Lib/concurrent/futures/process.py

```diff
@@ -660,9 +665,12 @@ def map(self, fn, *iterables, timeout=None, chunksize=1):
                               timeout=timeout)
         return _chain_from_iterable_of_lists(results)
 
-    def shutdown(self, wait=True):
+    def shutdown(self, wait=True, cancel_futures=False):
```

Comment:
Maybe new arguments should be keyword-only - there is at least one other PR out now aiming to add an argument to `shutdown`.

Reply:
Sounds good, I'm in favor of making *cancel_futures* keyword-only.